### PR TITLE
Side Navbar

### PR DESCRIPTION
**Pull request by AmrMohamed5**
Add a side navbar

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -20,6 +20,9 @@
       integrity="sha384-KyZXEAg3QhqLMpG8r+8fhAXLRk2vvoC2f3B09zVXn8CA5QIVfZOJ3BCsw2P0p/We"
       crossorigin="anonymous"
     />
+    <!-- Font Awsome -->
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
```
Comment (majidzeno) [inline public/index.html]: Try to add it by NPM

File: public/index.html [hunk 2]
```diff
@@ -20,6 +20,9 @@
...
+
 
     <title>React App</title>
```
Comment (majidzeno) [inline public/index.html]: We need to change this to Fox

File: public/index.html (hunk 1 above)
Comment (AmrMohamed5) [reply public/index.html]: Resolved

File: public/index.html (hunk 2 above)
Comment (AmrMohamed5) [reply public/index.html]: Resolved

**Review by majidzeno — APPROVED** [review]

**Pull request merged**